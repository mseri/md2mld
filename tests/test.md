# this is a markdown test document

This is _important_ or **very important**, let's try the `code` fences.

```
let x = "bla"
```

Hello this is a ordered list

1. One
2. Two
3. Three

And this is a unordered list

- One
- Two
- Three

## Another header

Blah blah blah

> This is some quote
>> This is a nested quote

And this is a link [Hello](https://ocaml.org).

    another code block?

<https://ocaml.org> <-- A link

<module:Module> <-- An explicit cross reference

<ref:Module> <-- An inferred cross reference


| foo   | bar     | tut   |
|:----- | ------- | -----:|
| _baz_ | bim     | `who` |
| per   | **lol** | dit   |

## Code

``` python
def f():
    return 0
```
