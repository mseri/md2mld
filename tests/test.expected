{0 this is a markdown test document}

This is {e important} or {b very important}, let's try the [code] fences.

{[
let x = "bla"
]}

Hello this is a ordered list
{ol {+ One
}{+ Two
}{+ Three
}}

And this is a unordered list
{ul {- One
}{- Two
}{- Three
}}

{1 Another header}

Blah blah blah
{%html: 
<blockquote>
This is some quote

<blockquote>
This is a nested quote
</blockquote>
</blockquote>
%}
And this is a link {{: https://ocaml.org} Hello}.

{[
another code block?
]}

{{: https://ocaml.org} https://ocaml.org} <-- A link

{!module:Module} <-- An explicit cross reference

{!Module} <-- An inferred cross reference

{table
{tr {th foo}{th bar}{th tut}}
{tr {td {e baz}}{td bim}{td [who]}}
{tr {td per}{td {b lol}}{td dit}}
}

{1 Code}

{@python[
def f():
    return 0
]}
